comps/dataprep/opensearch/README.md

```diff
@@ -0,0 +1,255 @@
+# Dataprep Microservice with OpenSearch
+
+For dataprep microservice for text input, we provide here the `Langchain` framework.
+
+## 🚀1. Start Microservice with Python（Option 1）
+
+### 1.1 Install Requirements
+
+- option 1: Install Single-process version (for 1-10 files processing)
+
+```bash
+apt update
+apt install default-jre
+apt-get install tesseract-ocr -y
+apt-get install libtesseract-dev -y
+apt-get install poppler-utils -y
```

Comment:
Would you mind linking me to the OPEA style guidelines because I'm confused as to what the standards are. This snippet is identical to existing code:

![image (34)](https://github.com/user-attachments/assets/af273966-1703-4b4f-8f08-a778a4213a37)